tests/test_init.py

```diff
@@ -256,6 +256,7 @@ def generate_params():
     xfail = {
         "pushapk-firefox-dev",
         "pushapk-firefox-fake-prod",
+        "pushapk-firefox-prod",
```

Comment:
Curious what the problem with these is :)

Reply:
Me too! It wasn't just one problem, of course. Fixed now.

Reply:
Actually, it seemed fixed locally, but wasn't reliable in all environments. I found *another* problem - pushapk's init_worker relies on keytool - so I restored the xfails and added a comment.